h2o-algos/src/main/java/hex/glm/GLMUtils.java

```diff
@@ -365,4 +365,19 @@ public static Frame buildRIDFrame(GLMModel.GLMParameters parms, Frame train, Fra
       train.add(parms._response_column, responseVec);
     return train;
   }
+  
+  public static boolean notZeroLambdas(double[] lambdas) {
+    if (lambdas == null) {
+      return false;
+    } else {
+      return ((int) Arrays.stream(lambdas).filter(x -> x != 0.0).boxed().count()) > 0;
+    }
+  }
+  
+  public static List<String> predictorChange(ComputationState state, List<String> coeffNames) {
```

Comment:
It looks like this method is not used anymore.

Reply:
Removed.